javascriptv3/example_code/bedrock-runtime/action/invoke-claude.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fileURLToPath } from "url";
+
+import { BedrockRuntimeClient, InvokeModelCommand } from "@aws-sdk/client-bedrock-runtime";
+
+/**
+ * @typedef {Object} ResponseBody
+ * @property {string} completion
+ */
+
+/**
+ * Invokes the Anthropic Claude 2 model to run an inference using the input
+ * provided in the request body.
+ *
+ * @param {string} prompt - The prompt that you want Claude to complete.
+ * @returns {string} The inference response (completion) from the model.
+ */
+export const invokeClaude = async (prompt) => {
+    const client = new BedrockRuntimeClient( { region: 'us-east-1' } );
+
+    const modelId = 'anthropic.claude-v2';
+
+    /* Claude requires you to enclose the prompt as follows: */
+    const enclosedPrompt = `Human: ${prompt}\n\nAssistant:`;
+
+    /* The different model providers have individual request and response formats.
+     * For the format, ranges, and default values for Anthropic Claude, refer to:
+     * https://docs.anthropic.com/claude/reference/complete_post
+     */
+    const payload = {
+        prompt: enclosedPrompt,
+        max_tokens_to_sample: 500,
+        temperature: 0.5,
+        stop_sequences: [ '\n\nHuman:' ],
+    };
+
+    const command = new InvokeModelCommand({
+        body: JSON.stringify(payload),
+        modelId: modelId,
```

Comment:
```suggestion
        modelId,
```

Reply:
https://ui.dev/shorthand-properties

Reply:
Thanks for your suggestion about using JavaScript shorthand properties! I usually lean towards property: value pairs for consistency, especially when at least one property doesn't share the same name. I find this approach enhances readability, which is particularly valuable in code examples. However, I understand the importance of following documentation standards, so I'm more than willing to use shorthand properties as recommended.

Reply:
It's fine either way. Just educating in case it was new info.